README.md

```diff
@@ -78,6 +78,10 @@ cp .env.example .env
 
 Open the `.env` file in your editor and change what it is needed, usually they are the first section of the file.
 
+**Important:** If this is not used for local development, please: 
```

Comment:
Maybe add this way

For local development update the following configuration for the caddy
- Update administrative endpoint to  `admin 0.0.0.0:2019` in `Caddyfile` here<link>
- Expose port `2019`  of `caddy`  in `docker-compose.yml` here <link>